pkg/config/auth.go

```diff
@@ -0,0 +1,79 @@
+package config
```

Comment:
This file probably should not be named `auth.go`, this is more related to doing get requests, right?

Reply:
Yeah, Would `client.go` be better here?

Reply:
Have renamed it! 🙂